task4/index.js

```diff
@@ -0,0 +1,24 @@
+var arr = [ 
+    {value: 100, type: 'USD'}, 
+    {value: 215, type: 'EUR'}, 
+    {value: 7, type: 'EUR'}, 
+    {value: 99, type: 'USD'}, 
+    {value: 354, type: 'USD'}, 
+    {value: 12, type: 'EUR'}, 
+    {value: 77, type: 'USD'}, ];
+var sumOfUsd = 0;
+
+var objArr = []
+
+for (let obj of arr) {

```

Comment:
This implementation is fine but the purpose of this task is to give a better understanding of array methods and functional programming in js. So please try to add new implementation with the usage of filter(), map() and reduce()